go/apps/api/routes/v2_keys_verify_key/403_test.go

```diff
@@ -26,30 +26,32 @@ func TestForbidden(t *testing.T) {
 	h.Register(route)
 
 	workspace := h.Resources().UserWorkspace
-	rootKey := h.CreateRootKey(workspace.ID, "api.*.verify_key")
-
-	headers := http.Header{
-		"Content-Type":  {"application/json"},
-		"Authorization": {fmt.Sprintf("Bearer %s", rootKey)},
-	}
 
 	api := h.CreateApi(seed.CreateApiRequest{WorkspaceID: workspace.ID})
 	key := h.CreateKey(seed.CreateKeyRequest{
 		WorkspaceID: workspace.ID,
 		KeyAuthID:   api.KeyAuthID.String,
 	})
 
-	t.Run("wrong api id", func(t *testing.T) {
+	t.Run("root key with wrong api permissions", func(t *testing.T) {
```

Comment:
this test is in the wrong file now I guess
tbh doing 1 file per status for this endpoint doesn't really work well anyways, idk what we should do here